cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt

```diff
@@ -39,16 +39,14 @@ parameter "param_cost_metric" do
   type "string"
   label "Cost Metric"
   allowed_values "Unamortized Unblended","Amortized Unblended","Unamortized Blended","Amortized Blended"
-  default "Unamortized Unblended"
+  default "Amortized Unblended"
   description "Select the cost metric for your report.  See the README file for more details"
 end
 
-##
 parameter "param_dimensions" do
   type "list"
   label "Cost Anomaly Dimensions"
-  description "Dimensions to run anomaly report. Unable to take in custom tags as a dimension"
-  allowed_values "Cloud Vendor","Cloud Vendor Account Name","Cloud Vendor Account","Manufacturer Name","Spend Type","Flexera Capability","Category","Instance Type","Region","Operation","Resource Type","Service","Billing Account ID","Usage Unit","Usage Type","Line Item Type","Purchase Option","Resource Group","Bill Entity","Bill Source","Invoice ID"
+  description "Add please the exact case sensitive name(s) or id(s) of Default or Rule Based Dimension you wish to run anomaly report. e.g. 'CCO Environments' for name / e.g. 'rbd_cost_center' for id. Also you can add more filters for anomalies on specific dimension by adding 'Vendor=AWS'; first criteria will be mandatory for consequent filters"
```

Comment:
Can we change to ?
Add please the exact case sensitive name(s) or id(s) of Default or Rule Based Dimension you wish to run anomaly report. e.g. 'CCO Environments' for name / e.g. 'rbd_cost_center' for id. Also you can add more filters for anomalies on specific dimension by adding 'Vendor=AWS'; first criteria will be mandatory for consequent filters